R/constructNWISURL.R

```diff
@@ -319,7 +319,8 @@ constructNWISURL <- function(siteNumbers,
 #' retrieval for the earliest possible record.
 #' @param endDate character ending date for data retrieval in the form YYYY-MM-DD. Default is "" which indicates
 #' retrieval for the latest possible record.
-#' @param legacy Logical. If TRUE, uses legacy WQP services. Default is FALSE.
+#' @param legacy Logical. If TRUE, use legacy WQP services. Default is TRUE.
```

Comment:
```suggestion
#' @param legacy Logical. If TRUE, uses legacy WQP services. If FALSE, uses WQX3.0 WQP services. Default is TRUE.
```

Reply:
Or maybe you don't want to offer that at this point.

Reply:
That text is basically in the next line. I do think changing from use to uses sounds better though.